lint-file.js

```diff
@@ -0,0 +1,17 @@
+const path =require('path');
+
+const kebabCase = /^[a-z0-9]*(?:[a-z0-9+-])+[a-z0-9]+$/;
+
+module.exports = function lintFile(file) {
+  const extensionName = path.extname(file);
+  const basename = path.basename(file, extensionName);
+  const isValid = kebabCase.test(basename);
+  const error = !isValid
+    ? { message: `${file} has wrong name.` }
+    : undefined;
+
+  return {
+    isValid,
```

Comment:
Isn't it redundant? If `error` is defined, the file can't be valid. Returning `Error` or `null` would be enough I think.

Reply:
done